### ISSUE TITLE
Beautiful job!

### ISSUE THREAD

**Issue by marcusletric** [contentful/contentful-management.js#140]
I was wondering if I can use content-preview API with the client, then I just followed the source to the core api and I found out that I just have to supply the host parameter in the config object when I create the client.... Amazing job guys, all this source is so clean and with good SOC!

**Comment by axe312ger** [comment]
First of all, thank you very much, Marcus. 🙇  Providing clear, readable code which allows easy contribution is one of our main goals in our public projects :)

Could it be that you are misusing the management SDK to get content for your app? You should use the delivery SDK to do this. It also mentions in the readme how to use the preview api with it. https://github.com/contentful/contentful.js#using-this-sdk-with-the-preview-api

**Comment by marcusletric** [comment]
Thank you for heads up! 
I am using the delivery SDK client with content delivery token and I want to provide a preview of draft content on another part of the site. I was going upon the API reference meanwhile I missed that part of the repo READ.ME so I started digging in the source to find out how I can configure the preview API host.

I used the management SDK previously for editing purposes as well, that was also a really nice tool to work with. It was clicking perfectly with the angular 4 project.
